Symfony/CS/Tests/Fixtures/Integration/priority/empty_return,useless_return,whitespacy_lines,extra_empty_lines.test

```diff
@@ -0,0 +1,45 @@
+--TEST--
```

Comment:
Please move this test to misc.
Fixers in `misc` could check priority as well.
`priority` dir is one to follow relations between 2 fixers defined in `FixerTest.php`